### PR TITLE
Improve contrast for filelist, form inputs

### LINKED ISSUE #3635: [Visual requirement – Make Code – Report Abuse] Luminosity ratio is less than 3:1 for border line of edit fields under 'Report Abuse' pop-
**User Experience:**
Visual impaired user will find difficulty to see the focus indicator over the control or element and border line for edit field if that focus indicator and border line have insufficient color contrast ratio as user won't be able to proceed further.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Report Abuse" option and select it.
7. Navigate through all the controls appearing under "Report Abuse" dialog.
8. Verify contrast ratio for border line of (What is the URL of the offensive project & Why do you find it offensive) edit fields.
.
**Actual Result:**
Luminosity ratio is less than 3:1 for border line of (What is the URL of the offensive project & Why do you find it offensive) edit fields under 'Report Abuse' pop-up and due to this, visual impaired user is unable to see the border line.

**Expected Result:**
Luminosity ratio should be more than and equal to 3:1 for border line of (What is the URL of the offensive project & Why do you find it offensive) edit fields under 'Report Abuse' pop-up, so that visual impaired user could easily see the border line.

**Note:**
Same issue exists on Firefox & Chrome.

**WCAG Reference:**
[WCAG 1.4.11 - Non-text Contrast
](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: CCA

### LINKED ISSUE #3640: [Visual requirement – Make Code – Explorer] Luminosity ratio is less than 3:1 for 'Add & Delete' icons present under 'Explorer' button.
**User Experience:**
Visual impaired user will find difficulty to see the non text control if that control or element have insufficient color contrast ratio as user won't be able to proceed further.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Java Script" drop down and select it.
6. Navigate to "Python" option and select it.
7. Navigate to "Explorer" and select it.
8. Navigate through all the controls after expanding Explorer control.
9. Verify contrast ratio of 'Add & Delete' icons by CCA tool.

**Actual Result:**
Luminosity ratio is less than 3:1 for 'Add & Delete' icons present under 'Explorer' button and due to this, visual impaired user is unable to see that icon.

**Expected Result:**
Luminosity ratio should be more than and equal to 3:1 for 'Add & Delete' icons present under 'Explorer' button, so that visual impaired user could easily such icon.

**Note:**
Same issue exists on Firefox & Chrome.

**WCAG Reference:**
[WCAG 1.4.11 - Non-text Contrast
](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: CCA

### PR DESCRIPTION
**Pull request by livcheerful**
fixes #3640

from #999999 to #8f8f8f
Before:
image

After:
image

fixes #3635 #3593 #3556 
Before:
image

After
image

Before: (focused)
image

After:
image

### CODE REVIEW

File: theme/style.less [hunk 1]
```diff
@@ -106,15 +106,20 @@
...
         img {
-            margin-bottom:1rem;            
+            margin-bottom:1rem;
         }
     }
+}
+
+/* Filelist */
+.ui.vertical.menu.filemenu .item>.label {
+    background-color: #8F8F8F;
```
Comment (pelikhan) [inline theme/style.less:124]: Is this microbit only or can we fix this in pxt for all editors?

Comment (livcheerful) [reply theme/style.less:124]: Oh I wasn't sure if other editors had styles that I shouldn't mess with -- I can put it in pxt for a better default styling! Are there editors you have in mind that I should test with?

Comment (pelikhan) [reply theme/style.less:124]: arcade :)

Comment (livcheerful) [reply theme/style.less:124]: haha ezpz I'll make the change!

**Comment by livcheerful** [comment]
new pr up at https://github.com/microsoft/pxt/pull/8058 !

**Pull request closed without merging**